apps/OpenSpace/main.cpp

```diff
@@ -995,6 +995,57 @@ void setSgctDelegateFunctions() {
     sgctDelegate.setStatisticsGraphScale = [](float scale) {
         sgct::Engine::instance().setStatsGraphScale(scale);
     };
+    sgctDelegate.setMouseCursor = [](WindowDelegate::Cursor mouse) {
+        GLFWwindow* w = glfwGetCurrentContext();
+        GLFWcursor* cursor;
+        switch (mouse) {
+        case WindowDelegate::Cursor::Arrow: {
+            cursor = glfwCreateStandardCursor(GLFW_ARROW_CURSOR);
```

Comment:
It might be more efficient to create a list of all of these cursors once at startup in this cpp file and then we point to the created ones?   The way it is right now, we'll recreate a new cursor every time it changes, leaking memory and losing performance without need.

So having a :
```cpp
std::unordered_map<WindowDelegate::Cursor, GLFWcursor*> Cursors = {
    { WindowDelegate::Cursor::Arrow, glfwCreateStandardCursor(GLFW_ARROW_CURSOR) },
    { WindowDelegate::Cursor::IBeam, glfwCreateStandardCursor(GLFW_IBEAM_CURSOR) },
   etc
};
```
higher up, then this function would become:
```cpp
GLFWwindow* w = glfwGetCurrentContext();
auto cursor = Cursors.find(mouse);
if (cursor != Cursors.end()) {
    glfwSetCursor(w, *cursor);
}
else {
    glfwSetCursor(w, nullptr);
```